### PR TITLE
Clearify config folder above 4.8

### PR DESCRIPTION
**Pull request by Tastaturberuf**
Should we write it more detailed?

### CODE REVIEW

**Comment by Tastaturberuf** [comment]
maybe without the first slash?

File: docs/manual/system/settings.de.md [hunk 1]
```diff
@@ -237,7 +237,7 @@ mehr Logging im Debug Modus). Außerdem committest du die `config.yml` im Gegens
 [Repository](https://de.wikipedia.org/wiki/Repository). Ein Repository kannst du verwenden, um deine Projekt-Versionen abzulegen, z. B. mit Git.
 
 {{% notice note %}}
-Ab der Version **4.8** von Contao befindet sich die Datei im Ordner `config`.
+Ab der Version **4.8** von Contao befindet sich die Datei im direkt Wurzelverzeichnis der Installation unter `/config`.
```
Comment (fritzmg) [inline docs/manual/system/settings.de.md]:
```suggestion
Ab der Version **4.8** von Contao befindet sich die Datei direkt im Wurzelverzeichnis der Installation unter `config/`.
```

**Comment by fritzmg** [comment]
I don't know if that is really necessary. The same would have to be done in the first paragraph of that section.

**Comment by netzarbeiter** [comment]
Thank you, Daniel

**Pull request merged**